ti/config/config.go

```diff
@@ -0,0 +1,69 @@
+package config
+
+import "github.com/harness/lite-engine/ti/client"
+
+type Cfg struct {
+	client       *client.HTTPClient
+	sourceBranch string
+	targetBranch string
+	commitBranch string
+	dataDir      string
+}
+
+func NewCfg(endpoint, token, accountID, orgID, projectID, pipelineID, buildID, stageID, repo, sha, commitLink,
```

Comment:
Rename it to New since it is present in config package

Reply:
Done